spec/eventboss/configuration_spec.rb

```diff
@@ -16,6 +16,26 @@
     it 'caches evaluated default' do
       expect(subject.sns_client.object_id).to eq(subject.sns_client.object_id)
     end
+
+    context 'when in ENV' do
```

Comment:
I would add test case checking `TRUE` and `FALSE` as we downcase values.

Reply:
@kbiedrzycki added